packages/js-example-app/src/index.js

```diff
@@ -1,15 +1,16 @@
 import '@boiler/ui-library/dist/index.js';
 import './style.scss';
+import {componentsPrefix} from '../../../prefix';
 
 const toggleLoadingButton = document.querySelector('#toggleLoadingState');
 const toggleDisabledState = document.querySelector('#toggleDisabledState');
 const logsContainer = document.querySelector('#logs');
 
-const blrButton = document.getElementsByTagName('blr-text-button')[0];
-const blrCheckbox = document.getElementsByTagName('blr-checkbox')[0];
-const blrSelect = document.getElementsByTagName('blr-select')[0];
-const blrTextInput = document.getElementsByTagName('blr-text-input')[0];
-const blrTextArea = document.getElementsByTagName('blr-textarea')[0];
+const button = document.getElementsByTagName(`${componentsPrefix}-text-button`)[0];
```

Comment:
couldnt we use the already computed tag names here, from the render functions?
I.e. ButtonGroupTagName.

Maybe we should even include them in out ui-library/src/index.ts as exported variable?